src/nginx/HISTORY.rst

```diff
@@ -3,9 +3,9 @@
 Release History
 ===============
 
-0.1.1
+1.0.0
 ++++++
-* Update the GA sku in the creation example.
+* Deployment identity, logging and scaling added. Configuration analyze added.
 
```

Comment:
Is it an add-on to this HISTORY.rst, not replacing old one?

Reply:
It is not replacing the old one. The version generated by codegen was 0.1.1. I updated the version as per the versioning schema.